theseus/geometry/functional/so3.py

```diff
@@ -658,3 +665,40 @@ def backward(cls, ctx, grad_output):
 _jproject_autograd_fn = None
 
 project = lie_group.UnaryOperatorFactory(_module, "project")
+
+
+# -----------------------------------------------------------------------------
+# Left Act
+# -----------------------------------------------------------------------------
+def _left_act_impl(group: torch.Tensor, matrix: torch.Tensor) -> torch.Tensor:
+    if not check_group_tensor(group):
+        raise ValueError("Invalid data tensor for SO3.")
```

Comment:
I just realized this raise error need to appear in a lot of place. Maybe it will be cleaner to move it inside the `check_group_tensor` function and do not return any bool back? The you only need to do:
`check_group_tensor(group)`

Reply:
+1